functions/src/api/lib/firebaseAuth.ts

```diff
@@ -0,0 +1,24 @@
+import {getAuth} from 'firebase-admin/auth';
+import {Context, Next} from 'koa';
+
+const firebaseAuth = () => async (ctx: Context, next: Next) => {
+  const {headers} = ctx;
+
+  if (headers?.authorization) {
+    // deny unsupported token format or empty token
+    const [type, token] = headers.authorization.split(' ', 2);
+    if ((type != 'Bearer' && type != 'bearer') || !token) {
```

Comment:
Nit pick, the linter must be borked here, should be `!==`

Reply:
True! And this is code from the old app... interesting 🤔